start_docs_server.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# convient script to run the docs server. It automatically rebuilds and restarts when you change code. You just need to refresh your browser. 
+WATCH_PATH="."
+START_SERVER_CMD="make develop-docs-comprehensive"
+
+while true; do
+  $START_SERVER_CMD &
+  SERVER_PID=$!
```

Comment:
I would guess this is not capturing the PID of the web server, but rather the `make` process, which launches the web server as a sub-process.  Which is maybe why you don't use this later?

Reply:
yes exactly